controller/src/test/java/io/pravega/controller/server/eventProcessor/ConcurrentEventProcessorTest.java

```diff
@@ -75,38 +92,40 @@ public PositionInternal asImpl() {
         }
     }
 
-    ConcurrentEventProcessor<TestEvent, TestRequestHandler> processor;
-    int checkpoint = -1;
-    Map<Integer, Boolean> map = new HashMap<>();
-    CompletableFuture<Void> latch = new CompletableFuture<>();
-    CompletableFuture<Void> result = new CompletableFuture<>();
+    private int checkpoint = -1;
+    private Map<Integer, Boolean> map = new HashMap<>();
+    private CompletableFuture<Void> latch = new CompletableFuture<>();
+    private CompletableFuture<Void> result = new CompletableFuture<>();
```

Comment:
Result, Latch and Map are only used by one test, and if another test were to use them, the two tests might mess with one another. Either move these 4 into local variables inside the test and make TestRequestHandler an inner class of the test. Or create a new private class that encompasses these 4 variables and pass it to the constructor of TestRequestHandler and keep a reference to it in the test.